src/preprocessor/noop.js

```diff
@@ -16,16 +16,13 @@ module.exports = {
   preprocess: null,
```

Comment:
what calls `registerParsedFile`?

Reply:
the gts parser. It's described at the top of this file